guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -46,7 +46,12 @@ There can only be one active job with a given purpose at a time.
 Click *Next* after you have entered the required information.
 . Review job details.
 You have the option to return to any part of the job wizard and edit the information.
-. Click *Run* to schedule the job for execution.
+. Click *Submit* to execute the job.
+
+When rerunning an existing job, the following buttons are also available:
```

Comment:
The buttons are always there, it doesn't matter if you're rerunning a job or running a brand new one.